cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -176,6 +181,67 @@ public void testDeleteSuspendedWorkflow() {
     Assert.assertTrue(store.getRuns(programId2, ProgramRunStatus.ALL, 0, Long.MAX_VALUE, Integer.MAX_VALUE).isEmpty());
   }
 
+  @Test
+  public void testWorkflowNodeState() throws Exception {
+    String namespaceName = "namespace1";
+    String appName = "app1";
+    String workflowName = "workflow1";
+    String mapReduceName = "mapReduce1";
+    String sparkName = "spark1";
+
+    ProgramId mapReduceProgram = new ProgramId(namespaceName, appName, ProgramType.MAPREDUCE, mapReduceName);
```

Comment:
same here and the next line - use the new EntityId syntax ``Ids.namespace(namespaceName)....``